### ISSUE TITLE
SRSecurityPolicy.m:11:9 'SRPinningSecurityPolicy.h' file not found & 'SRDelegateController.h' file not found

### ISSUE THREAD

**Issue by jtaoufik** [facebookincubator/SocketRocket#692]
<img width="1359" alt="image" src="https://github.com/user-attachments/assets/b0345f07-3919-4327-989b-81268358cd5a">

I got this issue after upgrading to latest macos version sequoia, could anyone help me on that ? :)

<img width="695" alt="image" src="https://github.com/user-attachments/assets/e9a218a0-fcb0-49cb-a152-cf8f2c9e1d67">
<img width="953" alt="image" src="https://github.com/user-attachments/assets/f40c2445-417e-4a9a-a6e4-f635199c43ad">

**Comment by jtaoufik** [comment]
they seem both present in the project hierarchy, I don't understand...
I got this issue since last week, tried multiple ways but none worked.

**Comment by cipolleschi** [comment]
Hi @jtaoufik. I'm sorry you are facing this issue.
Some questions to get more context:
1. Was it working on Sonoma? 
2. Which version of Xcode are you using?
 3. Are you opening the `XXX.xcworkspace` file or the `XXX.xcodeproj` file? After running pod install, you should use the `XXX.workspace`

**Comment by jtaoufik** [comment]
Yes only in sonoma with xCode 16, I reverted back to xcode 15.2 and executed it using command line (binary).

I don't understand why it happened !

**Comment by nastasiupta** [comment]
Any updates on that?

**Comment by nastasiupta** [comment]
https://media0.giphy.com/media/v1.Y2lkPTc5MGI3NjExaGowNzZ0N2NwMzRleWp6Zm9zN2puaWI4MjNrd3lxZ3ZpZHh0YThyZiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/PCvkgunX9ZbEEyfTQH/giphy.gif
